js/modules/timeline.src.js

```diff
@@ -271,83 +461,85 @@ seriesType('timeline', 'line', {
                     enabled: false
                 }
             };
-        Series.prototype.bindAxes.call(this);
-        extend(this.xAxis.options, timelineXAxis);
-        extend(this.yAxis.options, timelineYAxis);
-    },
-    translate: function () {
-        var series = this,
-            points = series.points,
-            xMap;
-
-        if (points) {
-            xMap = series.mappedXData = this.getMappedXData();
-
-            // Apply new calculated point values.
-            each(points, function (point, i) {
-                point.applyOptions({
-                    x: xMap[i]
-                });
-            });
-        }
-        Series.prototype.translate.call(this);
+        Series.prototype.bindAxes.call(series);
+        extend(series.xAxis.options, timelineXAxis);
+        extend(series.yAxis.options, timelineYAxis);
     }
 }, {
     init: function () {
-        var args = arguments,
-            point = Point.prototype.init.apply(this, args),
-            xMap = args[0].mappedXData;
-
-        point.applyOptions({
-            x: xMap[args[2]],
-            y: 1
-        });
+        var point = Point.prototype.init.apply(this, arguments);
+        point.name = pick(point.name, point.date, 'Event');
+        point.y = 1;
 
         return point;
     },
+    setVisible: Pie.prototype.pointClass.prototype.setVisible,
```

Comment:
I think that better is define own method, because in custom-builder we will include the full pie chart, but use one method. Its not efficient.